mesonbuild/compilers/compilers.py

```diff
@@ -147,6 +147,14 @@ def is_library(fname):
                       'custom': [],
                       }
 
+ccrx_buildtype_args = {'plain': [],
+                       'debug': ['-debug'],
+                       'debugoptimized': ['-debug'],
```

Comment:
We are transitioning away from having these flags in build type args. Instead they should come from the `debug` option only. Search for `clike_debug_args` to see how it should be done. Eventually we hope to remove buildtype arguments altogether by moving all of them to individually togglable options (build types remain as a shorthand for "enable X, Y and Z".

Reply:
Sure no problem. I'll add a `ccrx_debug_args` but we'll keep `ccrx_buildtype_args` still?

Reply:
Those should be empty, or not have `-debug` in them at least. See `gnulike_buildtype_args` for example.